portal/src/locale-data/en.json

```diff
@@ -1577,6 +1577,9 @@
   "SMSProviderConfigurationScreen.provider.twilio": "Twilio",
   "SMSProviderConfigurationScreen.provider.webhook": "Webhook",
   "SMSProviderConfigurationScreen.provider.deno": "Custom JS/TS",
+  "SMSProviderConfigurationScreen.form.twilio.twilioSID": "Application SID",
+  "SMSProviderConfigurationScreen.form.twilio.twilioAuthToken": "AuthToken",
```

Comment:
Reword the label to `Auth Token` (note the space). Again this is the official name. https://www.twilio.com/docs/iam/api/authtoken